features/thamos_advise.feature

```diff
@@ -22,6 +22,10 @@ Feature: Running thamos advise against deployment
 
         Examples: Advise
             | git_repo                                                             | runtime_environment     | user_stack  | static_analysis |
-            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | download_dataset        | without     | without         |
+            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | download-dataset        | with        | without         |
```

Comment:
Isn't it `overlays/` missing in all URL to user_stack?

e.g. `https://github.com/thoth-station/elyra-aidevsecops-tutorial/overlays`

Reply:
The URL is pointing to a Git repo to be cloned. The runtime environment is picked based on the configuration in `.thoth.yaml` respecting `runtime_environment` name stated in the `.thoth.yaml`.

Reply:
we can remove test-model then, as .thoth.yaml file has only `download-dataset`, `training` and `inference` runtime environments.